packages/wdio-static-server-service/tests/launcher.test.ts

```diff
@@ -64,5 +64,5 @@ test('should register middlewares', async () => {
 })
 
 afterEach(() => {
-    express.mockClear()
+    (express as unknown as jest.Mock).mockClear()
```

Comment:
Is it necessary to use `as` twice?

Reply:
If not using "as unknown", i am getting the below error:

`Conversion of type 'typeof e' to type 'Mock<any, any>' may be a mistake because neither type sufficiently overlaps with the other. If this was intentional, convert the expression to 'unknown' first`